mlfinlab/tests/test_trend_scanning_labels.py

```diff
@@ -37,7 +38,7 @@ def test_trend_scanning_labels(self):
 
         # Before 2008/5/12 we had a strong positive trend
         self.assertTrue(
-            set(tr_scan_labels.loc[pd.Timestamp(2008, 1, 1):pd.Timestamp(2008, 5, 9)].bin) == set([1]))
+            set(tr_scan_labels.reindex(pd.date_range(pd.Timestamp(2008, 1, 1), pd.Timestamp(2008, 5, 9))).dropna().bin) == set([1]))
```

Comment:
We use reindex instead of loc because of Pandas warning?

Reply:
I use reindex to be compatible with pandas > 1.0.